### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by ride90**
- github actions
- fix linter/pytest errors
- fix reportlab version to have support for python 3.5

### CODE REVIEW

**Review by petrjasek — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      - name: cache npm
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm
+            ${{ runner.os }}-
+
+      - run: npm install --no-audit
```
Comment (petrjasek) [inline .github/workflows/tests.yml:65]: if you have package-lock you can use `npm ci` should be a bit faster

Comment (ride90) [reply .github/workflows/tests.yml:65]: ohh right

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 httmock
 wooper
 PyRTF3
+# reportlab is fixed due to support for python 3.5
+reportlab==3.5.53
```
Comment (MarkLark86) [inline requirements.txt:5]: This makes sense in the `master` branch, when this is ported to the `develop` we probably should bump Python to 3.6+

**Review by MarkLark86 — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (marwoodandrew) [reply requirements.txt:5]: This one caught us out on our last release!

**Pull request merged**